examples/configurations/flink-carry-over-offset-between-statements/README.md

```diff
@@ -0,0 +1,53 @@
+### Notes
+
+1. Make sure to use to create a Flink table called `customers_source` before running this example.
```

Comment:
Maybe you meant?
```suggestion
1. Make sure to use or create a Flink table called `customers_source` before running this example.
```

Reply:
That's a great catch! I'll fix it in a separate PR 🫡

Reply:
update: fixed it in this PR 🫡